src/alexa_go_vote/polling_place.cljs

```diff
@@ -76,21 +70,24 @@
 (defn process-response
   [request response]
   (if (http/unexceptional-status? (:status response))
-    (let [body (response->clj response)]
+    (let [body (response-body->clj (:body response))]
       (if (contains? body :pollingLocations)
         (first-polling-place-response body)
         (if (str/blank? (get-in request [:intent :slots :state :value] nil))
-          (request-state request)
+          (if (= "COMPLETED" (:dialogState request))
```

Comment:
This is a pretty confusing pile of conditionals. I think this one could be collapsed into the one right above it, since they are both going to end up in `no-info-response` on the `else` case, right? Otherwise, would it make sense to split any of this out into a separate function simply for clarity's sake?